opentech/apply/funds/serializers.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import serializers
+
+from opentech.apply.activity.models import Activity
+from .models import ApplicationSubmission, Round
+
+
+class SubmissionListSerializer(serializers.ModelSerializer):
+    title = serializers.CharField(source='page.title')
+
+    class Meta:
+        model = ApplicationSubmission
+        fields = ('id', 'title', 'status')
+
+
+class SubmissionDetailSerializer(serializers.ModelSerializer):
```

Comment:
We have ha to write some of this for the work we have been doing.

See below for how we serialised the questions/meta

https://github.com/OpenTechFund/opentech.fund/pull/852/files#diff-7ba15010bea2ee21506ce1d29441cafe